### DISCUSSION TITLE
Using bantime increment how

### DISCUSSION THREAD

**Discussion by hobbycharken** [fail2ban/fail2ban#3880] [Q&A] [answered]
I have an iredmail mailserver running with fail2ban installed. I would like to use bantime increment and have some questions
if I enable it, shall I take away the settings for bantime, findtime, maxretry?
Can I set it up so after f.ex 5 failes, the ip get banned for good?

Regards

Anders Yuran

**Comment by sebres** [comment, marked as answer]
> shall I take away the settings for bantime, findtime, maxretry?

No, they are still needed, however possibly adjusted, for instance bantime becomes initial (duration of first ban) and so can be reduced maybe, in order to reduce a banning time by an accidental ban for legitimate users, e. g. on false positive. Also one could increase maxretry (together with findtime).
The incremental banning if enabled would enlarge the (initial) bantime as well as reduce attempt count by further attempts of known as bad IPs, see the table in #1460.

> Can I set it up so after f.ex 5 failes, the ip get banned for good?

Sure one can, but... Persistent banning is not recommended (unneeded load on net-filter side by lot of IPs, especially considering that IPs may be dynamic, etc).
Vice versa it is recommended to setup `bantime.maxtime` for example to 5w (5 weeks) - an IP reaching this bantime will be banned immediately by first attempt, so it can make maximal 10 attempts per year - what is simply much too little for a successful bruteforce attack, even if it is distributed.

**Reply by hobbycharken** [reply]
Thankyou for your advice. Quite new to fail2ban so all is needed. I will take your advice and adjust settings.
I try to have max security on the email server so the users can have a stable service

Thanks again
